woocommerce/woocommerce
# Fix menu not being pinned correctly with certain screen widths

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #41511.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Adjust your browser's width to around 980 (I calculated this by typing `window.innerWidth` in the console)
2. Zoom to 125%
3. Enable the new product editor
4. Go to Product > Add
5. Scroll to the bottom of the page
6. Verify that the WordPress menu keeps pinned and doesn't scroll along



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
